src/api-reference/user-provisioning/v4.user-provisioning.md

```diff
@@ -40,31 +78,34 @@ Name|Description|Endpoint
 `identity.user.core.read`|Read user core data.|GET
 `identity.user.coresensitive.read`|Read core sensitive data.|GET
 `identity.user.enterprise.read`|Read user enterprise data.|GET
-`travel.user.general.read`|Read general Travel data: Manager, Travel Name, RuleClass, Groups, OrgUnit, Travel Custom Fields|GET
-`travel.user.private.read`|Read private Travel data. XML Sync ID, CRSName/GDS Sync ID, Travel Name Remark, Gender|GET
+`travel.user.general.read`|Read general Travel data.|GET
+`travel.user.private.read`|Read private Travel data.|GET
 `spend.user.general.writeonly`|Change spend user information.|POST, PUT, PATCH
 `spend.user.general.read`|View spend user information.|GET
 
-## Dependencies <a name="dependencies"></a>
+## <a name="dependencies"></a>Dependencies
 
-SAP Concur users must purchase either Concur Expense or Concur Travel or both in order to use this API. This API only available to approved early adopter partners and clients. Please contact your SAP Concur representative for more information.
+SAP Concur users must purchase either Concur Expense or Concur Travel or both in order to use this API. This API is not available in the China Data Center.  Please contact your SAP Concur representative for more information.
 
-## Access Token Usage <a name="access-token-usage"></a>
+## <a name="access-token-usage"></a>Access Token Usage
 
 This API supports company level access tokens.
 
-## Events <a name="event-usage"></a>
+## <a name="event-usage"></a>Events
```

Comment:
Is there a UPS Event already built? if so, we will need Event documentation to add to the site as well. If it is not already created, we cannot include it into this current ref.

Reply:
Hi, it is already created.